extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -152,14 +152,6 @@ class AmpYoutube extends AMP.BaseElement {
     this.playerReadyPromise_ = deferred.promise;
     this.playerReadyResolver_ = deferred.resolve;
 
-    // TODO(aghassemi, #3216): amp-youtube has a special case where 404s are not
-    // easily caught hence the following hacky-solution.
-    // Please don't follow this behavior in other extensions, instead
-    // see BaseElement.createPlaceholderCallback.
```

Comment:
Is this comment no longer valid?

Reply:
This component is still using a standard image element instead of an `amp-img`. Considering we intend to deprecate AMP Image, it made sense to not try and use it in a new place.

Reply:
So we don't care about the 404s if an `img` is used instead?

Reply:
I'm not following, the code as written handles the 404 case (and the fake 404 case).

Reply:
I'm just interested in knowing if the initial workaround that avoided `createPlaceholderCallback` is no longer necessary.

Reply:
Yes, the implementation now works (and passes tests for all known cases).